includes/faveo-pricing-functions.php

```diff
@@ -1,400 +1,260 @@
 <?php
+// Detect user IP
+function fhai_get_user_ip() {
+    $ip = '';
+
+    if ( isset( $_SERVER['HTTP_CLIENT_IP'] ) && ! empty( $_SERVER['HTTP_CLIENT_IP'] ) ) {
+        $ip = sanitize_text_field( wp_unslash( $_SERVER['HTTP_CLIENT_IP'] ) );
+    } elseif ( isset( $_SERVER['HTTP_X_FORWARDED_FOR'] ) && ! empty( $_SERVER['HTTP_X_FORWARDED_FOR'] ) ) {
+        $ipList = explode( ',', sanitize_text_field( wp_unslash( $_SERVER['HTTP_X_FORWARDED_FOR'] ) ) );
+        $ip     = trim( $ipList[0] );
+    } elseif ( isset( $_SERVER['REMOTE_ADDR'] ) && ! empty( $_SERVER['REMOTE_ADDR'] ) ) {
+        $ip = sanitize_text_field( wp_unslash( $_SERVER['REMOTE_ADDR'] ) );
+    }
+
+    return $ip;
+}
+
+
+// Get country from IP using ipapi.co (secure HTTPS)
+function fhai_get_country_from_ip($ip) {
+    // Fallback for localhost testing
+    if ($ip === '127.0.0.1' || $ip === '::1') {
```

Comment:
localhost can be many ip not just this, The logic should be like if the api is not able to find an ip default to a value. This can be done with try catch i feel